feat(content): Umbral Reach Update (#10298)
endless-sky/endless-sky: data/rulei/rulei.txt

			`	It's best you don't remain here for too long.`
				decline

mission "Rulei: You Are (Not) Alone"
	invisible
	landing
	to complete
		never
	to offer
		has "First Contact: Rulei: offered"
	on enter "L-118"
		log `Discovered a region of emptiness east of the core, where systems are shrouded by a vast, eerie darkness. Other than dim stars and a few planets, nothing exists.`
		conversation
			`Upon arriving into this system, you notice a strange murkiness curtaining the stars beyond, and your scanners are having trouble making sense of it. The few stellar objects present are eerily blotted out by a vast, nebulous shade. Aside from them, it's as if there is nothing here. Even the shade itself is nothing more than the emptiness of space, and throughout it, no one. Nothing.`

	on enter "L-6181"
		fail
		conversation

> 			`As you continue through this empty space, you come across nothing but planets of silence and diminished stars. However, upon entering this system, you are met with what appears to be a creature of some kind. Strangely, your scanners don't read it as any sort of lifeform, categorizing it only as a mere "collection" of matter. It blends with space itself, as if it were simply a reflection of the darkness beyond.`

Reviewer: I personally think this sounds better, but up to you.
```suggestion
			`As you continue through this empty space, you come across nothing but diminished stars and planets of silence. However, upon entering this system, you are met with what appears to be a creature of some kind. Strangely, your scanners don't read it as any sort of lifeform, categorizing it only as a mere "collection" of matter. It blends with space itself, as if it were simply a reflection of the darkness beyond.`
```

> 			`As you continue through this space of emptiness, you come across nothing but planets of silence and diminished stars. However, upon entering this system, you are met with what appears to be a creature of some kind. Strangely, your scanners don't read it as any sort of lifeform, categorizing it only as a mere "collection" of matter. It blends with space itself, as if it were simply a reflection of the darkness beyond.`

Reviewer: ```suggestion
			`As you continue through this empty space, you come across nothing but planets of silence and diminished stars. However, upon entering this system, you are met with what appears to be a creature of some kind. Strangely, your scanners don't read it as any sort of lifeform, categorizing it only as a mere "collection" of matter. It blends with space itself, as if it were simply a reflection of the darkness beyond.`
```

> 			`As you continue through this space of emptiness, you come across nothing but planets of silence and diminished stars. However, upon entering this system, you are met with what appears to be a creature of some kind. Strangely, your scanners don't read it as any sort of lifeform, rather, it appears as some sort of "collection" of matter. It blends with space itself, as if it were simply a reflection of the darkness beyond.`

Reviewer 2: I like the flavor of this conversation, but there's some internal inconsistency. 
If the player's scanners don't read the Jje as a lifeform, how do they know to describe it as a "creature"? Does it approach the ship such that FL can see it with their own eyes, and thus confirm its living nature? i.e.
```suggestion
			`As you continue through this space of emptiness, you come across nothing but planets of silence and diminished stars. However, upon entering this system, you catch a glimpse of what appears to be a creature of some kind. Strangely, your scanners don't read it as any sort of lifeform, rather, it appears as some sort of "collection" of matter. It blends with space itself, as if it were simply a reflection of the darkness beyond.`
```
A "'collection' of matter" is equally nebulous: presumably the sensors describe _anything with mass_ as a collection of matter. If it's their inability to describe it that you want to emphasize, maybe something like the following would better convey that?
```suggestion
			`As you continue through this space of emptiness, you come across nothing but planets of silence and diminished stars. However, upon entering this system, you are met with what appears to be a creature of some kind. Strangely, your scanners don't read it as any sort of lifeform, categorizing it only as a mere "collection" of matter. It blends with space itself, as if it were simply a reflection of the darkness beyond.`
```

Author: The latter sounds good. As for it being "a creature", I wouldn't put it past the same kind of recognition the player has for something like void sprites, or presumably the Vyrmeid. It's still "appears to be", as it's a moving form of sorts amongst dead systems, so I don't think it's too far fetched.

	npc
		government "Indigenous Lifeform (Acheron)"

> 		personality timid unconstrained uninterested mute

Reviewer 3: ```suggestion
		personality mute timid unconstrained uninterested
```

		has "First Contact: Rulei: offered"
		has "visited system: L-118"
	on enter "Kanguwa"
		event "antumbral reach label"

> 		log "Factions" "Rulei" `The Rulei seem to acknowledge the void of systems east of the core as the "Antumbral Reach", and appear to have some sort of emotional connection to the region. Other than stating to "beware" the area, they will not further comment on it.`

Reviewer 3: ```suggestion
		log "Factions" "Rulei" `The Rulei seem to acknowledge the void of systems east of the core as the "Antumbral Reach," and appear to have some sort of emotional connection to the region. Other than stating to "beware" the area, they will not further comment on it.`
```

Reviewer 3: For some reason this didn't get resolved?

Reviewer: The original line was right, the comma is supposed to be outside of the quotation.

		conversation
			`Out of nowhere, the mysterious whispers of the Rulei seep into your mind once more. They murmur slowly at first, but grow faster and louder from the direction of the ancient vessel, eventually coalescing into one spoken phrase.`
			`	"Beware..."`

> 			`	"...Umbral Reach.."`

Reviewer 3: ```suggestion
			`	"...Umbral Reach..."`
```
